src/Generator.ts

```diff
@@ -67,6 +67,10 @@ export default class Generator {
 
     const arrivalTime = departureTime.plus({ hours: duration.hours, minutes: duration.minutes }).setZone(destination.timezone);
 
+    // cost is calculated as a percentage of distance, between 10% and 30%
+    // this allows for variability in flight prices while maintaining some relativity between values
+    // (i.e., longer flights are usually more expensive)
+    const randCost = distance * (this.random(10,30) / 100);
```

Comment:
```suggestion
    const randCost = distance * (this.random(10, 30) / 100);
```

Reply:
Hey Amanda! We're still waiting on an answer from legal before we can merge this. Sorry for the delay.
Everything else looks good except this one issue with a missing space here. ;)